symfony/plugins/orangehrmAdminPlugin/test/Service/SkillServiceTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Admin\Tests\Service;
+
+use OrangeHRM\Admin\Dao\SkillDao;
+use OrangeHRM\Admin\Dto\SkillSearchFilterParams;
+use OrangeHRM\Admin\Service\SkillService;
+use OrangeHRM\Config\Config;
+use OrangeHRM\Entity\Skill;
+use OrangeHRM\Tests\Util\TestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+
+/**
+ * @group Admin
+ */
+class SkillServiceTest extends TestCase
+{
+
+    private $skillService;
+    protected $fixture;
+
+    /**
+     * Set up method
+     */
+    protected function setUp(): void
+    {
+        $this->skillService = new SkillService();
+        $this->fixture = Config::get('sf_plugins_dir') . '/orangehrmAdminPlugin/test/fixtures/SkillDao.yml';
+        TestDataService::populate($this->fixture);
+    }
+
+    public function testDeleteSkill(): void
+    {
+        $toBeDeletedSkillIds = array(1, 2);
```

Comment:
refer [this](https://stackoverflow.com/questions/23697248/phpstorm-replace-array-notation-to-short-syntax) to convert array() to []